app/views/_account.html.erb

```diff
@@ -7,20 +7,20 @@
       </button>
       <ul class="dropdown-menu" role="menu">
         <li>
-          <%= link_to 'My Account', page_path('account') %>
+          <%= link_to 'Your Account', main_app.page_path('account') %>
```

Comment:
Set up the "Your account" label in the locale yaml to DRY it up. Probably a separate configuration than the 'blacklight.header_links.logout' one.